### ISSUE TITLE
Training Loop

### ISSUE THREAD

**Issue by krekiehn** [bingogome/samm#72]
Hello,

is there an option to retrain / fine-tune a given model to (manuel corrected) data?

Greetings Nicolai

**Comment by bingogome** [comment]
Yes, you can certainly do that. You could simply port the fine-tuned model file and replace the one in the workspace.
